cloud/aws/deploy/down.go

```diff
@@ -28,25 +28,6 @@ import (
 	"google.golang.org/grpc/status"
 )
 
-type DownStreamMessageWriter struct {
```

Comment:
Is deleting this intentional?

Reply:
these should be using the common message writer now